discovery-provider/src/queries/get_unpopulated_playlists.py

```diff
@@ -72,7 +75,8 @@ def get_unpopulated_playlists(session, playlist_ids):
     playlists_response = []
     for playlist_id in playlist_ids:
         if playlist_id in cached_playlists:
-            playlists_response.append(cached_playlists[playlist_id])
+            if not filter_deleted or not cached_playlists[playlist_id]['is_delete']:
+                playlists_response.append(cached_playlists[playlist_id])
```

Comment:
Does this mess up the ordering of our playlist response? Like if we request Ids [1, 2, 3] but 2 is deleted, now we're returning [Playlist1, Playlist3] and we don't know what corresponds to what? Or does order not matter?

Reply:
I use this in place of the actual query for tracks w/ these ids and the filter, so the same thing should happen that happened before.